components/automate-cli/pkg/adminmgmt/adminmgmt.go

```diff
@@ -162,8 +162,9 @@ func EnsureTeam(ctx context.Context,
 
 	if !found && !dryRun {
 		_, err := apiClient.TeamsClient().CreateTeam(ctx, &iam_req.CreateTeamReq{
-			Id:   id,
-			Name: name,
+			Id:       id,
+			Name:     name,
+			Projects: []string{},
```

Comment:
Why was this `projects` necessary? I would have expected it to be fine omitted

Reply:
it's ok to omit it from the HTTP request, it gets marshalled to `Projects: []string{}` at the grpc level. Since we're using the grpc API directly here we have to specify the empty array.

Reply:
(seems like we should be handling that at every level, though)